test/ProductConstructionService.ScenarioTests/TestHelpers.cs

```diff
@@ -133,7 +133,7 @@ public static async Task<string> Which(string command)
     public static string FormatExecutableCall(string executable, params string[] args)
     {
         var output = new StringBuilder();
-        var secretArgNames = new[] { "-p", "--password", "--github-pat", "--azdev-pat" };
+        var secretArgNames = new[] { "--github-pat", "--azdev-pat" };
```

Comment:
don't we also want to hide the Maestro token here?

Reply:
True

Reply:
```suggestion
        var secretArgNames = new[] { "-p", "--password", "--github-pat", "--azdev-pat" };
```